### PR TITLE
Fix links in readme file

### PR DESCRIPTION
**Pull request by mohsinr**
This pr fixes links for main readme file for internal token project, examples and releases page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,15 +3,14 @@
 ![Nuget](https://img.shields.io/nuget/v/Quantoz.Nexus.Sdk.Token)
 
 Welcome to the Quantoz Nexus SDK repository. In here you will find all the tools required to make your connection with Nexus even easier!
-
```
Comment (nagarwal4) [inline README.md:6]: Please undo this

**Review by nagarwal4 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -3,15 +3,14 @@
...
 ## Repository Overview
 
 This repository consists of the following projects:
 
-1. [Nexus Token SDK](./Nexus.Token.SDK) - This project contains functionalities required to interact with your Nexus Token environments.
```
Comment (nagarwal4) [inline README.md:11]: Also write about Nexus.Sdk.Shared

Comment (mohsinr) [reply README.md:11]: I have no idea what to write about Nexus Sdk Shared :) i just updated the existing links which were 404 :)

File: README.md (hunk 1 above)
Comment (mohsinr) [reply README.md]: Hi @nagarwal4  this is reverted https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/43/commits/b12c20b016bd0c228a064c3e64348b992ca2c932

**Review by nagarwal4 — APPROVED** [review]

**Pull request merged**